lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -293,6 +293,33 @@ sub _job_get($) {
     return $job->to_hash(assets => 1);
 }
 
+sub job_settings_subquery {
```

Comment:
I guess the memo was sent before you joined - but we declared Scheduler.pm as dead. Code should be removed from it, not added. So please put this code into the model.